NitroxServer/Server.cs

```diff
@@ -35,6 +35,13 @@ public void Start()
             Log.Info("Nitrox Server Started");
             EnablePeriodicSaving();
         }
+
+        public void Stop()
+        {
+            worldPersistence.Save(world);
```

Comment:
It would be proper to also stop the timer here that saves the world.

Can use `Save()` here too since it already exists. Same for the `timer.Elapsed+=`.